data-prepper-core/src/main/java/org/opensearch/dataprepper/pipeline/server/ListPipelinesHandler.java

```diff
@@ -41,6 +41,13 @@ public PipelineListing(final String name) {
 
     @Override
     public void handle(final HttpExchange exchange) throws IOException {
+        String requestMethod = exchange.getRequestMethod();
+        if (!requestMethod.equals("GET") && !requestMethod.equals("POST")) {
```

Comment:
Nit: We should use constant for those magic word. You might be able to find them out-of-the-box from HttpMethod.GET, HttpMethod.POST in javax

Reply:
Fixed in [145afa3](https://github.com/opensearch-project/data-prepper/pull/1697/commits/145afa3a4c81ef289ba188902961339ae35a2df3)